src/commands/components/versions.ts

```diff
@@ -22,6 +29,11 @@ export default class ComponentVersions extends Command {
     name: 'component_name',
   }];
 
+  static sensitive = new Set();
+
+  static non_sensitive = new Set([...Object.keys({ ...ComponentVersions.flags }),
```

Comment:
We could make a util function in BaseCommand that would handle this. Having to do this for each command would be hard to maintain.

Reply:
Yeah ideally most things here would be on the base command class

Reply:
Removed sensitive/non_sensitive sets, and added flag/arg property 'non_sensitive': boolean instead, which will be used to filter flag/arg metadata sent to Sentry from within the BaseCommand class.